src/shared/lib/HotspotSet.spec.ts

```diff
@@ -67,7 +73,13 @@ describe('HotspotSet', () => {
     });
 
     it('works with one region, consolidated from a few input regions', () => {
-        const hss = new HotspotSet([[4, 20], [0, 5], [2, 6], [-5, 0], [3, 3]]);
```

Comment:
https://prettier.io/blog/2019/11/09/1.19.0.html#break-arrays-of-arraysobjects-if-each-element-has-more-than-one-elementproperty-6694-by-sosukesuzuki